src/components/tricks/TrickDetails.js

```diff
@@ -202,14 +202,14 @@ const TrickDetails = () => {
           }
 
           <div className="row">
-            <h4>Set your success frequency:</h4>
+            <h4><Trans id="common.setSuccessFrequency" />:</h4>
             <div onChange={selectFreq}>
               <FreqList stickable={trick}/>
             </div>
           </div>
 
           <div className="boostSkill row justify-content-center">
-            <button className={trick.boostSkill ? "col-8 col-lg-3 col-xl-2 btn btn-warning" : "col-8 col-lg-3 col-xl-2 btn btn-primary" } onClick={toggleBoostSkill}>{trick.boostSkill ? "Unboost this trick" : (<><IoRocketSharp/> Boost this trick</>)}</button>
+            <button className={trick.boostSkill ? "col-8 col-lg-3 col-xl-2 btn btn-warning" : "col-8 col-lg-3 col-xl-2 btn btn-primary" } onClick={toggleBoostSkill}>{trick.boostSkill ? "Unboost this trick" : (<><IoRocketSharp/> <Trans id="trickDetails.boostThisTrick">Boost this trick</Trans></>)}</button>
```

Comment:
Also here we still need to wrap the "Unboost this trick" in a <Trans> tag

Reply:
Fixed in [334d334](https://github.com/bastislack/highline-freestyle/pull/198/commits/334d3346a6a6506656230c65abcaddba41153219)